certora/conf/rewardsCoordinator.conf

```diff
@@ -0,0 +1,28 @@
+{
+    "files": [
+        "lib/openzeppelin-contracts/contracts/token/ERC20/ERC20.sol",
+        "lib/openzeppelin-contracts/contracts/token/ERC20/presets/ERC20PresetFixedSupply.sol",
+        "certora/harnesses/RewardsCoordinatorHarness.sol"
+    ],
+    "verify": "RewardsCoordinatorHarness:certora/specs/core/RewardsCoordinator.spec",
+    "loop_iter": "2",
+    "parametric_contracts": ["RewardsCoordinatorHarness"],
+    "packages": [
+        "@openzeppelin=lib/openzeppelin-contracts",
+        "@openzeppelin-upgrades=lib/openzeppelin-contracts-upgradeable"
+    ],
+    "link": [],
+    "optimistic_fallback": true,
+    "optimistic_loop": true,
+    "optimistic_hashing": true,
+    "rule_sanity": "basic",
+    "exclude_rule": ["checkClaimNeverFalse"],
```

Comment:
so is this rule just failing to pass right now then?

Reply:
No the rule is called in paymentCoordinatorWithoutSanity.conf. The reason for the separate conf is that this rule fails the sanity check since checkClaim reverts instead of returning false.